contrib/ci-builders/tekton/tekton-labs/tasks/zcash-build-bench.yml

```diff
@@ -0,0 +1,205 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: zcash-build-bench
+spec:    
+  params:
+    - name: JOBS
+      default: "4"
+      type: string
+    - name: BUILD_FOR_HOST
+      default: x86_64-pc-linux-gnu
+      type: string
+    - name: CONFIGURE_FLAGS
+      default: ""
+      type: string
+    - name: BUILD_CONTAINER
+      default: electriccoinco/zcashd-bench-debian10
+      type: string
+    - name: TEST_TARGET
+      default: ""
+      type: string
+  resources:
+    inputs:
+      - name: source
+        type: git
+  steps:
+    - name: build
+      image: $(params.BUILD_CONTAINER)
+      script: |
+        #!/bin/bash
+        set -o pipefail
+        cd workspace/source
+        echo $(git rev-parse HEAD)
+        echo $(git rev-parse --short HEAD)
+        
+        if [ "$(params.BUILD_FOR_HOST)" == "x86_64-apple-darwin11" ]
+        then
+            mkdir -p depends/SDKs
+            curl -fs https://ecc.mypinata.cloud/ipfs/QmeSwckvSCGL9SXGdEHoAyqXdzD7T9HYTwsC34Bj5EVDF5 \
+              -o depends/SDKs/Xcode-11.3.1-11C505-extracted-SDK-with-libcxx-headers.tar.gz
+            tar zxvf depends/SDKs/Xcode-11.3.1-11C505-extracted-SDK-with-libcxx-headers.tar.gz \
+              -C depends/SDKs
+        fi              
+
+        CONFIGURE_FLAGS=$(params.CONFIGURE_FLAGS) HOST=$(params.BUILD_FOR_HOST) ./zcutil/build.sh -j$(params.JOBS)
+
+    - name: bench-sleep
+      image: $(params.BUILD_CONTAINER)
+      script: |
+        #!/bin/bash
+        set -o pipefail
+        python -m pip install pyblake2 pyflakes python-qpid-proton pyutil pyzmq requests simplejson
+        
+        cp -a /home/.zcash-params $HOME/
+        cp -r /home/benchmark-200k-UTXOs.tar.xz /workspace/source
+        cp -r /home/block-107134.tar.xz /workspace/source
+        cd /workspace/source
+
+        ./qa/zcash/performance-measurements.sh time sleep | jq  '[.[]|.runningtime | tonumber] | add / (. | length | tonumber)'
```

Comment:
This appears to hard-code the calculation of an average. The prior Buildbot benchmark collection computed multiple statistics that were better suited to benchmarking runtimes:
- Median
- Lower and upper quartiles
- Extrema (min and max values)

We should collect all of these (and the average as well if we want it around), so that we can get an idea of the distribution of benchmarked values (which is important for optimisation).